Tests/FoundationEssentialsTests/AttributedString/AttributedStringTests.swift

```diff
@@ -2599,4 +2599,26 @@ E {
         XCTAssertEqual(testConstrainedContainer.filter(inheritedByAddedText: true), AttributeContainer.testInt(2).testParagraphConstrained(3).testCharacterConstrained(4))
         XCTAssertEqual(testConstrainedContainer.filter(inheritedByAddedText: false), AttributeContainer.testNonExtended(5))
     }
+
+    func testWritingDirectionBehavior() {
+        // Indicate that this sentence is primarily right to left, because the English term "Swift" is embedded into an Arabic sentence.
+        var string = AttributedString("Swift مذهل!", attributes: .init().writingDirection(.rightToLeft))
+
+        // To remove the information about the writing direction, set it to `nil`:
+        string.writingDirection = nil
+
+        let range = string.range(of: "Swift")!
```

Comment:
small nit to avoid test crashes if a bug is introduced
```suggestion
        let range = try XCTUnwrap(string.range(of: "Swift"))
```